pkg/apis/operator/v1alpha1/glustercluster_types.go

```diff
@@ -4,19 +4,62 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// GlusterClusterCredentials defines secret credentials
+type GlusterClusterCredentials struct {
```

Comment:
Could this and `Credentials` in `glusternode_types.go` be named similarly?

Reply:
Ack